src/features/submitButton/SubmitButton.module.css

```
@@ -0,0 +1,17 @@
+.submitButton {
+  position: fixed;
+  bottom: 0;
```

Stylelint (17.11.0)

[error] Duplicate property "bottom" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)